README.md

```diff
@@ -1,4 +1,4 @@
 # monmon
 
 Run tests: `ruby -I lib test/process_test.rb`
-Ruby CLI: `./bin/cli`
+Ruby CLI: `./bin/cli -a DB/CSV`
```

Comment:
This is invalid command. So show two commands: `./bin/cli -a DB` and `./bin/cli -a CSV` with explanation what they do.